### PR TITLE
[HPriest] Holy priest talent section

### PR DESCRIPTION
**Pull request by TJMoats**
This PR adds a card for every holy priest talent, and a new section for organizing them. All talents have been moved out of the statistics section. 

https://puu.sh/Bx5MN/6ceab4d491.png

### CODE REVIEW

File: src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  statistic() {
+    return (
+
+      <StatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.APOTHEOSIS_TALENT.id} />}
+        value={"Value"}
+        label="Apotheosis"
+        tooltip={``}
+      />
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js]: This tooltip is empty.

File: src/Parser/Priest/Holy/Modules/Talents/100/HolyWordSalvation.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.HOLY_WORD_SALVATION_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.totalHealing} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/HolyWordSalvation.js]: No need for React.Fragment here.

File: src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  statistic() {
+    return (
+
+      <StatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.LIGHT_OF_THE_NAARU_TALENT.id} />}
+        value={"Value"}
+        label="Light of the Naaru"
+        tooltip={``}
+      />
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js]: Empty tooltip and "Value"

File: src/Parser/Priest/Holy/Modules/Talents/100/HolyWordSalvation.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+          Healing from Salv: ${formatThousands(this.healingFromSalv)}<br />
+          Healing from Renews: ${formatThousands(this.healingFromRenew)}<br />
+          Healing from PoMs: ${formatThousands(this.healingFromPom)}
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(7);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/HolyWordSalvation.js]: this type of statisticOrder is deprecated. Use the position={} instead.

File: src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(7);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(7);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/15/EnduringRenewal.js [hunk 7]
```diff
@@ -73,35 +75,24 @@ class EnduringRenewal extends Analyzer {
...
+        tooltip={`Refreshed Renews: ${this.refreshedRenews}`}
       />
+
     );
-    //
   }
-
-  statisticOrder = STATISTIC_ORDER.OPTIONAL(1);
+  statisticOrder = STATISTIC_ORDER.CORE(1);
 }
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/15/EnduringRenewal.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/15/Enlightenment.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.ENLIGHTENMENT_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemManaGained amount={this.enlightenmentMana} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/15/Enlightenment.js]: No need for React.Fragment here

File: src/Parser/Priest/Holy/Modules/Talents/15/Enlightenment.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+        )}
+        label="Enlightment"
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(1);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/15/Enlightenment.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/15/TrailOfLight.js [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.TRAIL_OF_LIGHT_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.totalToLHealing} />
+          </React.Fragment>
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/15/TrailOfLight.js]: No need for React.Fragment here.

File: src/Parser/Priest/Holy/Modules/Talents/15/TrailOfLight.js [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+        label="Trail of Light"
+        tooltip={`Trail of Light Procs: ${this.totalToLProcs}`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(1);
+}
+
+export default TrailOfLight;
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/15/TrailOfLight.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/30/AngelicFeather.js [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.ANGELIC_FEATHER_TALENT.id} />}
+        value={`${this.angelicFeatherCasts} Feather(s) cast`}
+        label="Angelic Feather"
+      />
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(2);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/30/AngelicFeather.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/30/AngelsMercy.js [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+        label="Angels Mercy"
+        tooltip={`
+          Desperate Prayers cast: ${this.desperatePrayersCast}<br />
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(2);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/30/AngelsMercy.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/30/Perseverance.js [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.PERSEVERANCE_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.totalDamageReduced} />
+          </React.Fragment>
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/30/Perseverance.js]: No need for React.Fragment here.

File: src/Parser/Priest/Holy/Modules/Talents/30/Perseverance.js [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+        tooltip={`
+          Preserverance Uptime: ${formatPercentage(this.uptime)}%<br />
+          Damage Reduced: ${formatThousands(this.totalDamageReduced)}
+          `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(2);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/30/Perseverance.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/45/Afterlife.js [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.AFTERLIFE_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.healingInAfterlife} />
+          </React.Fragment>
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/Afterlife.js]: No need for React.Fragment here

File: src/Parser/Priest/Holy/Modules/Talents/45/Afterlife.js [hunk 17]
```diff
@@ -0,0 +1,80 @@
...
+        label="Afterlife"
+        tooltip={`Extra Spirit of Redemption time: ${Math.floor(this.spiritOfRedemptionBonusTime / 1000)}s`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(3);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/Afterlife.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js [hunk 18]
```diff
@@ -38,17 +39,20 @@ class CosmicRipple extends Analyzer {
...
+          </React.Fragment>
+        )}
+        label="Cosmic Ripple"
       />
 
     );
   }
-  statisticOrder = STATISTIC_ORDER.CORE(5);
+  statisticOrder = STATISTIC_ORDER.CORE(3);
 }
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js [hunk 19]
```diff
@@ -38,17 +39,20 @@ class CosmicRipple extends Analyzer {
   statistic() {
     return (
 
-      <StatisticBox
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js]: This is a talent, but uses a TraitStatisticBox -- same goes for almost all the other talents aswell

File: src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js [hunk 20]
```diff
@@ -38,17 +39,20 @@ class CosmicRipple extends Analyzer {
...
         icon={<SpellIcon id={SPELLS.COSMIC_RIPPLE_HEAL.id} />}
-        value={this.owner.formatItemHealingDone(this.totalHealing)}
-        label="Cosmic Ripple Healing"
-        tooltip={`<strong>Total Healing:</strong> ${formatThousands(this.totalHealing)}<br>
-                  <strong>Total Overhealing:</strong> ${formatThousands(this.overhealing)}`}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.totalHealing} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js]: No need for React.Fragment here

File: src/Parser/Priest/Holy/Modules/Talents/45/GuardianAngel.js [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.GUARDIAN_ANGEL_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.guardianSpiritSelfHealing} />
+          </React.Fragment>
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/GuardianAngel.js]: No need for React.Fragment here

File: src/Parser/Priest/Holy/Modules/Talents/45/GuardianAngel.js [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+        tooltip={`
+          Total Guardian Spirits Cast: ${this.guardianSpiritCasts}<br />
+          Total Guardian Spirit Resets: ${this.guardianSpiritRefreshes}
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(3);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/45/GuardianAngel.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/60/Censure.js [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+          ${this.chastiseCasts} Chastise Casts<br />
+          ${this.censureStuns} Chastise Stuns<br />
+          ${this.censureIncomp} Censure Incapacitates
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(4);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/60/Censure.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/60/Censure.js [hunk 24]
```diff
@@ -0,0 +1,59 @@
...
+
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.CENSURE_TALENT.id} />}
+        value={(
+          `${this.censureStuns + this.censureIncomp} Censure CC(s)`
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/60/Censure.js]: Don't need the () here

File: src/Parser/Priest/Holy/Modules/Talents/60/PsychicVoice.js [hunk 25]
```diff
@@ -0,0 +1,47 @@
...
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.PSYCHIC_VOICE_TALENT.id} />}
+        value={`${this.psychicScreamHits} Targets Feared`}
+        label="Psychic Voice"
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(4);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/60/PsychicVoice.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/60/ShiningForce.js [hunk 26]
```diff
@@ -0,0 +1,47 @@
...
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.SHINING_FORCE_TALENT.id} />}
+        value={`${this.shiningForceHits} Knock Back(s)`}
+        label="Shining Force"
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(4);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/60/ShiningForce.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/75/BindingHeal.js [hunk 27]
```diff
@@ -0,0 +1,89 @@
...
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.BINDING_HEAL_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.bindingHealHealing} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/BindingHeal.js]: No need to use <React.Fragment> here

File: src/Parser/Priest/Holy/Modules/Talents/75/BindingHeal.js [hunk 28]
```diff
@@ -0,0 +1,89 @@
...
+          Casts:&#9;${this.bindingHealCasts}<br />
+          Self Healing:&#9;${formatThousands(this.bindingHealSelfHealing)} (${formatPercentage(this.getOverhealPercent(this.bindingHealSelfHealing, this.bindingHealSelfOverhealing))}% OH)<br />
+          Party Healing:&#9;${formatThousands(this.bindingHealPartyHealing)} (${formatPercentage(this.getOverhealPercent(this.bindingHealPartyHealing, this.bindingHealPartyOverhealing))}% OH)
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(5);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/BindingHeal.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/75/CircleOfHealing.js [hunk 29]
```diff
@@ -0,0 +1,74 @@
...
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.CIRCLE_OF_HEALING_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.circleOfHealingHealing} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/CircleOfHealing.js]: Don't need <React.Fragment> here

File: src/Parser/Priest/Holy/Modules/Talents/75/CircleOfHealing.js [hunk 30]
```diff
@@ -0,0 +1,74 @@
...
+          Coh Casts: ${this.circleOfHealingCasts}<br />
+          Total Healing: ${formatThousands(this.circleOfHealingHealing)} (${formatPercentage(this.overHealPercent)}% OH)<br />
+          Average Targets Hit: ${this.averageTargetsHit.toFixed(2)}
+        `}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(5);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/CircleOfHealing.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/75/SurgeOfLight.js [hunk 31]
```diff
@@ -0,0 +1,77 @@
...
+  statistic() {
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.SURGE_OF_LIGHT_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.solHealing} />
+          </React.Fragment>
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/SurgeOfLight.js]: Don't need React.Fragment here.

File: src/Parser/Priest/Holy/Modules/Talents/75/SurgeOfLight.js [hunk 32]
```diff
@@ -0,0 +1,77 @@
...
+        )}
+        tooltip={`${this.solFlashHeals} free Flash Heals`}
+        label="Surge of Light"
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(5);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/75/SurgeOfLight.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/90/Benediction.js [hunk 33]
```diff
@@ -0,0 +1,50 @@
...
+    return (
+
+      <TraitStatisticBox
+        category={STATISTIC_CATEGORY.TALENTS}
+        icon={<SpellIcon id={SPELLS.BENEDICTION_TALENT.id} />}
+        value={(
+          <React.Fragment>
+            <ItemHealingDone amount={this.healingFromBenedictionRenews} />
+          </React.Fragment>
+        )}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/90/Benediction.js]: Don't need <React.Fragment> here

File: src/Parser/Priest/Holy/Modules/Talents/90/Benediction.js [hunk 34]
```diff
@@ -0,0 +1,50 @@
...
+        label="Benediction"
+        tooltip={`${this.renewsFromBenediction} total Renews from Benediction`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(6);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/90/Benediction.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/90/DivineStar.js [hunk 35]
```diff
@@ -0,0 +1,67 @@
...
+          </React.Fragment>
+        )}
+        label="Divine Star"
+        tooltip={`Divine Star Cast: ${this.divineStarCasts}`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(6);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/90/DivineStar.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/90/Halo.js [hunk 36]
```diff
@@ -0,0 +1,66 @@
...
+          </React.Fragment>
+        )}
+        label="Halo"
+        tooltip={`Halos Cast: ${this.haloCasts}`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(6);
+}
```
Comment (Pewtro) [inline src/Parser/Priest/Holy/Modules/Talents/90/Halo.js]: this type of statisticOrder is deprecated. Use the position={} prop instead.

File: src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js (hunk 1 above)
Comment (TJMoats) [reply src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js]: The module is disabled. With as big as the PR is, I want to get in what I can and add the last 2 talents later on. Apotheosis is going to be a pita.

File: src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js (hunk 3 above)
Comment (TJMoats) [reply src/Parser/Priest/Holy/Modules/Talents/100/LightOfTheNaru.js]: Same as for Apotheosis.

File: src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js (hunk 1 above)
Comment (Pewtro) [reply src/Parser/Priest/Holy/Modules/Talents/100/Apotheosis.js]: Preferably just remove it completely for now then imo 😃

File: src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js (hunk 19 above)
Comment (TJMoats) [reply src/Parser/Priest/Holy/Modules/Talents/45/CosmicRipple.js]: The components would be the same. I can create a carbon copy of the existing one I can create one.

**Comment by TJMoats** [comment]
@Pewtro I believe I've made all of the requested changes.

**Comment by TJMoats** [comment]
@Pewtro I ended up finishing the rest of the talents and am opening a new PR as the scope has changed.

**Pull request closed without merging**